### PR TITLE
Catalog to resolver

### PR DESCRIPTION
**Pull request by saevar43**
# API Catalogue pages added to new link resolver

## What

Moved API Catalogue pages up a level, added them to the link resolver, and change navigation accordingly.

## Why
We want the catalogue pages to be accessible in both Icelandic and English.

## Screenshots / Gifs

N/A

## Checklist:

- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] Formatting passes locally with my changes
- [x] I have rebased against main before asking for a review

### CODE REVIEW

**Review by saevarma — DISMISSED** [review]
LGTM

**Review by saevarma — APPROVED** [review]
LGTM

File: apps/web/screens/ApiCatalogue/ApiCatalogue.tsx [hunk 1]
```diff
@@ -412,6 +400,11 @@ const ApiCatalogue: Screen<ApiCatalogueProps> = ({
               {data?.getApiCatalogue?.services.length > 0 && (
                 <GridContainer>
                   <ServiceList
+                    baseUrl={
+                      activeLocale === 'en'
+                        ? './webservices/'
+                        : './vefthjonustur/'
+                    }
```
Comment (stefanhar) [inline apps/web/screens/ApiCatalogue/ApiCatalogue.tsx:407]: If you are resolving to the url `/throun/vefthjonustur` you could do it like this:
```ts
const { linkResolver } = useLinkResolver() // at the top of this cmp
...
<ServiceList
  baseUrl={linkResolver('webservicespage').as} // linkResolver ensures this link is of right locale
```

File: apps/web/screens/ServiceDetails/ServiceDetails.tsx [hunk 2]
```diff
@@ -154,8 +145,8 @@ const ServiceDetails: Screen<ServiceDetailsProps> = ({
                       { title: n('linkIslandIsText'), href: n('linkIslandIs') },
                       { title: n('linkThrounText'), href: n('linkThroun') },
                       {
-                        title: n('linkServiceListText'),
-                        href: n('linkServiceList'),
+                        title: n('linkServicesText'),
+                        href: n('linkServices'),
```
Comment (stefanhar) [inline apps/web/screens/ServiceDetails/ServiceDetails.tsx:149]: Using linkResolver for this link might remove the need for this namespace variable:
```ts
const { linkResolver } = useLinkResolver() // at the top of this cmp
...
{
  title: title: n('linkServicesText'),
  href: linkResolver('webservicespage').as
```
Also passing href as an object here might allow the cmp using this data to have prefetch by using the returned `as` and `href` properties

**Review by stefanhar — APPROVED** [review]

**Pull request merged**